src/integrationTest/java/uk/gov/hmcts/darts/common/controller/CourthouseApiTest.java

```diff
@@ -38,27 +56,120 @@ class CourthouseApiTest extends IntegrationBase {
         "tests/CourthousesTest/courthousesPostEndpoint/requestBody400_MissingCourthouseDisplayName.json";
     private static final String REQUEST_BODY_TEST_JSON = "tests/CourthousesTest/courthousesPostEndpoint/requestBodyTest.json";
 
+    private static final String ORIGINAL_USERNAME = "James Smith";
+    private static final String ORIGINAL_EMAIL_ADDRESS = "james.smith@hmcts.net";
+    private static final String ORIGINAL_DESCRIPTION = "A test user";
+    private static final boolean ORIGINAL_SYSTEM_USER_FLAG = false;
+    private static final OffsetDateTime ORIGINAL_LAST_LOGIN_TIME = OffsetDateTime.of(2023, 10, 27, 22, 0, 0, 0, ZoneOffset.UTC);
+    private static final OffsetDateTime ORIGINAL_LAST_MODIFIED_DATE_TIME = OffsetDateTime.of(2023, 10, 27, 22, 0, 0, 0, ZoneOffset.UTC);
+    private static final OffsetDateTime ORIGINAL_CREATED_DATE_TIME = OffsetDateTime.of(2023, 10, 27, 22, 0, 0, 0, ZoneOffset.UTC);
+
     @Autowired
     private MockMvc mockMvc;
 
+    @Autowired
+    private AdminUserStub adminUserStub;
+
+    @MockBean
+    private UserIdentity mockUserIdentity;
+
     @Test
     void courthousesGet() throws Exception {
+        UserAccountEntity user = adminUserStub.givenUserIsAuthorised(mockUserIdentity);
+        createEnabledUserAccountEntity(user);
+
         Integer addedId = addCourthouseAndGetId(REQUEST_BODY_HAVERFORDWEST_JSON);
 
-        MockHttpServletRequestBuilder requestBuilder = get("/courthouses/{courthouse_id}", addedId)
+        MockHttpServletRequestBuilder requestBuilder = get("/admin/courthouses/{courthouse_id}", addedId)
             .contentType(MediaType.APPLICATION_JSON_VALUE);
-        mockMvc.perform(requestBuilder).andExpect(status().isOk())
+        MvcResult response = mockMvc.perform(requestBuilder).andExpect(status().isOk())
             .andExpect(jsonPath("$.courthouse_name", is("HAVERFORDWEST")))
             .andExpect(jsonPath("$.code", is(761)))
             .andExpect(jsonPath("$.created_date_time", is(notNullValue())))
-            .andExpect(jsonPath("$.last_modified_date_time", is(notNullValue())));
+            .andExpect(jsonPath("$.last_modified_date_time", is(notNullValue())))
+            .andDo(print())
+            .andReturn();
+
+        assertEquals(200, response.getResponse().getStatus());
+        assertTrue(response.getResponse().getContentAsString().contains("security_group_ids"));
+        assertFalse(response.getResponse().getContentAsString().contains("region_id"));
+
+        verify(mockUserIdentity).userHasGlobalAccess(Set.of(ADMIN));
+        verifyNoMoreInteractions(mockUserIdentity);
+
+    }
+
+    @Test
+    @Transactional
```

Comment:
Please remove @Transactional from the tests

Reply:
When @Transactional is removed the following error is thrown:
- I get org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role 
This also occurs if fetch =  FetchType.EAGER is set

If trying to set dartsDatabase.save(courtHouseEntity);  the error thrown is:
- org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: [uk.gov](http://uk.gov/).hmcts.darts.common.entity.CourthouseEntity.regions: could not initialize proxy - no Session

After further discussion we have agreed to resolve this issue at a later date.